### DISCUSSION TITLE
Modified lines only support 9+ in treeview

### DISCUSSION THREAD

**Discussion by KrabsWong** [microsoft/vscode-discussions#115] [Extension Development QnA] [answered]
<img width="354" alt="image" src="https://user-images.githubusercontent.com/12368943/187346421-424deeec-0aa7-4269-a360-c5f65b946f17.png">

Like the screenshot above, even I have modified more than 10 lines in `package.json`, it just shows 9+.

Is there any way to show up the specific line number?

**Comment by alefragnani** [comment, marked as answer]
Hi @KrabsWong ,

I guess you mean _problems in file_, as defined by `"problems.decorations.enabled"` setting, not _modifications_, right?

If that's the case, I suppose you cannot show the specific number, as it is hard coded here https://github.com/microsoft/vscode/blob/1895178d0009566b2ae04a58cd9a9a51e125eff8/src/vs/workbench/contrib/markers/browser/markersFileDecorations.ts#L50

You probably need to open a feature request for such functionality.

Hope this helps
